RO-2998: Støtte 16kb minnesider på Android - forsøk 2

gruble opened the pull request:
Det er vanskelig å sjekke lokalt om vi støtter Android-kravet, men iflg. Copilot var det cordova-sqlite-storage som muligens var synderen. Har oppdatert denne + alle @awesome-cordova-plugins.
Etter jeg gjorde det fikk jeg problemer med å bygge, fordi denne importen feilet (i warning.service):
```ts
import { HTTP } from '@awesome-cordova-plugins/http/ngx';
```
Jeg fant ingen referanse til denne plugin'en i package.json, så skjønner ikke hvordan det har virket før, men etter tips fra @jorgkv testet jeg https://capacitorjs.com/docs/apis/http, og det så ut til å funke bra. Får nå hentet regioner fra iskart.no.

- [x] Hvis ikke jeg finner en bedre måte å sjekke om vi støtter 16kB-kravet lokalt, kan det være vi bare må publisere en ny versjon til Google Play og se om de godtar den. Iflg. [APK Analyzer i Android Studio](https://developer.android.com/guide/practices/page-sizes#identify-native-code) har vi to native biblioteker, libsqlcipher.so og libsqlc-ndk-native-driver.so. Jeg får ingen feilmeldinger på disse nå:
<img width="673" height="257" alt="image" src="https://github.com/user-attachments/assets/064fa693-acae-4ed9-8aee-5eb2b3fdfa91" />

gruble committed:
Skrur på pagedMemory i Capacitor

gruble committed:
Oppdatert cordova-plugins

gruble committed:
Byttet fra å bruke native http over alt til å kun bruke det for isvarsel

jorgkv commented on src/app/core/services/warning/warning.service.ts:
```diff
@@ -650,18 +649,19 @@ export class WarningService {
-      {
-        'Content-Type': 'application/json',
+    try {
+      const response = await CapacitorHttp.get({ url, headers: { 'Content-Type': 'application/json' } }).catch(
```
Du trenger kanskje ikke catch her, siden det er en ytre try catch uansett? Eller så skjønte jeg ikke helt hvorfor den var der i alle fall - legg gjerne på en kommentar.

gruble replied:
Det har du helt rett i. Bra sett.

gruble committed:
Fjernet unødvendig catch

gruble committed:
Byttet compileSdkVersion og targetSdkVersion til 35 for Cordova

Er usikker på om dette faktisk brukes lengre, men endrer det for sikkerhets skyld

gruble committed:
Fikset test som feilet

jorgkv commented on package-lock.json:
Tror det ble glemt å ta npm install når versjonsnr ble oppdatert sist, denne fila har endringer som ikke er relatert til saken. Det går sikkert greit.

gruble replied:
Det har du rett i. Jeg har gjort det nå og rebaset denne greina på develop
